conda/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "jina" %}
-{% set version = "2.0.23" %}
+{% set version = "2.1.5" %}
```

Comment:
how is this adapted? conda versions will only exist for this hardcoded versions? we do not have any job to update this?

Reply:
This is acutally auto-updated in the [feedstock repo](https://github.com/conda-forge/jina-feedstock) by conda-forge bot (see [this PR](https://github.com/conda-forge/jina-feedstock/pull/8) for example. The recipe here is not what is used by conda (you have to copy/paste it to feedstock repo first)

I could add some cd action here, but I feel it is not worth the effort - from 2.0.x till now I never had to make any adjustments, and I don't foresee this being a common thing. Right now I just need to make manual changes when there are changes in the requirements.
 
 What's really important is the script that generates the output.